scaling_performance/optimizing_storage.adoc

```diff
@@ -175,13 +175,64 @@ to nodes to support a healthy cluster.
 == Choosing a Docker Graph Driver
 
 Docker stores images and containers in a graph driver (a pluggable storage
-technology), such as `DeviceMapper`, `Overlay`, and `Btrfs`. Each have advantages and
-disadvantages. For example, `Overlay` is faster than `DeviceMapper` at starting and
-stopping containers, but is not Portable Operating System Interface for Unix (POSIX) compliant because of the architectural
-limitations of a union file system, and does not yet support SELinux.
+technology), such as `DeviceMapper`, `Overlay`, and `Btrfs`. Each have
+advantages and disadvantages. For example, `Overlay` is faster than
+`DeviceMapper` at starting and stopping containers, but is not Portable
+Operating System Interface for Unix (POSIX) compliant because of the
+architectural limitations of a union file system, and does not yet support
+SELinux.
+
+For more information about `Overlay`, including supportability and usage
+caveats, see the
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/7.3_Release_Notes/index.html#technology_previews_file_systems[Red
+Hat Enterprise Linux (RHEL) 7 Release Notes].
+
+.Docker Graph Driver Comparisons
+|===
+|Name |Description |Benefits |Limitations |Defaults
+
+|Device Mapper loop-lvm
+|Uses the device mapper thin provisioning module (dm-thin-pool) to implement copy
+on write (CoW) snapshots. For each device mapper graph location, thin pool is
+created based on two block devices, one for data and one for metadata. By
+default, these block devices are created automatically by using loopback mounts
+of automatically created sparse files.
+|Benefits go here
+|Limitations go here
```

Comment:
Limitations:  not all POSIX features work (i.e. O_DIRECT).  Most importantly, this mode is Unsupported for production workloads.

Reply:
Also it cannot be resized without destroying and re-creating the pool.  The 100GB default size is shared among all containers and images.